cLIMS/organization/migrations/0001_initial.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've read in that django book - that updating the whole thing by the mean of migrations is a good thing ...

But I think @nandankita wasn't using them before - I wonder if that would break anything ? @Snehal1894 did you test it locally ?

Reply:
i think it has to do with the state of the database, right ? i.e. `models` describe how the underlying database is organized

Reply:
Yes. And our code on git hub doesn't have these files. They are automatically updated according to our changes (or we generate new migration files in case there are changes wherever we are running the code). It is better to do it on server I think. So we don't need to add these type of files

Reply:
ok then I think we should remove them ...